README.md

```diff
@@ -108,43 +108,92 @@ See [CONFIG.md](CONFIG.md) for details.
 See [Code Samples](https://github.com/hazelcast/hazelcast-nodejs-client/tree/master/code_samples)
 
 
-# Serialization Considerations
+# Serialization
+**Table of Contents**
```

Comment:
For the sake of simplicity, I would rather not include the in-section TOC here. And maybe an introductory paragraph talking about serialization itself and why it is needed would be nice.